### ISSUE TITLE
Implement data backups

### ISSUE THREAD

**Issue by obs4qqf** [thecourseforum/theCourseForum2#665]
**Is your feature request related to a problem? Please describe.**
To avoid future issues with data loss, it would be a good idea to create some sort of automated data backup script that will save our production DB on DigitalOcean. This will ensure that user data is properly stored and accessible regardless of any issues that might arise.

**Describe the solution you'd like**
It would be best to create an automated script that will run periodically and backup our production data. Ideally, the script would run multiple times a semester (and definitely once at the end) to ensure we have multiple versions of the database at various points.

**Describe alternatives you've considered**
The backups probably should not be done within DigitalOcean itself to ensure that they aren't impacted by issues with have with the service. Also, if the automation part is difficult to figure out, then just the creation of a backup script should be fine for now.

**Additional context**
N/A

**Comment by wkaisertexas** [comment]
Why can't we just use Digital Ocean backups? In theory, vendor lock could be a problem and Digital Ocean could go down, but maintaining a custom backup system seems like a pain.

[Base Digital Ocean Backups](https://docs.digitalocean.com/products/images/backups/)

**Comment by barrett-ruth** [comment]
https://docs.digitalocean.com/support/can-i-download-a-backup-or-snapshot/

**Comment by wkaisertexas** [comment]
<img width="722" alt="Screenshot 2024-01-28 at 6 52 16 PM" src="https://github.com/thecourseforum/theCourseForum2/assets/27795014/1a4fe81b-5646-48fa-bfcf-650d19eb04f8">

tCF fits into the 500 mB snapshot free tier or you could just use SFTP.

**Comment by barrett-ruth** [comment]
Don't close an issue unless there is an implemented solution. I agree with you but there's no point in acting like it's done when it's not - let's resolve this next meeting.

**Comment by wkaisertexas** [comment]
I was joking for the last close. But, I think let's keep it simple and not really worry too much about custom backups and things like that.

**Comment by barrett-ruth** [comment]
Yeah, but our entire db got GG'ed over the summer. Although it was human error, well, that pretty much means we can rely on it happening again. I think some sort of backup system/protective measure is worth it - losing all of our reviews *cannot happen again*.

**Comment by wkaisertexas** [comment]
Wait, the entire legacy DB got killed? What?

**Comment by wkaisertexas** [comment]
Wait, didn't we get some of the data back? I am looking at reviews and we have ~30,000 since 2005

**Comment by barrett-ruth** [comment]
I think we lost the reviews over just the summer period - we had the backup before that. Either way, that totaled to >400 reviews I believe.
